ts-sdk/src/generated/index.ts

```diff
@@ -0,0 +1,33 @@
+/**
```

Comment:
I added this file manually for now, @acbeni, since I saw thats the new structure you use too 👍